src/main/resources/api.yaml

```diff
@@ -438,6 +438,30 @@ paths:
           description: User do not have permissions to get quiz results
         '404':
           description: Quiz not found
+  /quizzes/{quizId}/results/{numberOfTopResults}:
```

Comment:
I would change that to ` /quizzes/{quizId}/results`

Reply:
I approved, but consider this change. Not sure if this is some other RESTful convention but I really don't like that xD